Dockerfile

```diff
@@ -1,21 +1,35 @@
-FROM php:7.1-fpm
+FROM php:7.2-fpm
 
-# Based on https://github.com/docker-library/wordpress/blob/d122aa204948aecd651af435fe4ca058bb99545d/php7.1/fpm/Dockerfile
+# Based on https://github.com/docker-library/wordpress/blob/master/php7.2/fpm/Dockerfile
```

Comment:
Linking to master seems worse than the actual commit hash, why did you choose to do this?

Reply:
Laziness.